### ISSUE TITLE
Templates, Templates, Templates

### ISSUE THREAD

**Issue by ColonelPanics** [openflighthpc/flight-metal#30]
The `dhcpd.flight` should not be an internally generated template. Currently the file that is generated is very specific to our test use-case and should, instead, be made more general and be directly editable (possibly via a cluster-level edit/modify command - #29).

_**Yes, this will require some primitive templating ability in Flight Metal but it's already somewhere buried internally to generate that file so it's just moving it a bit further towards content**_

The file itself will only need to contain the bare minimum node configuration. Things like the gateway IP are specific to the tested use-case and were included erroneously (my bad!).

So the following is what it would need to be (pseudo-codey loop inbound):
```
<< for every node with a mac do >>
host NODENAME {
  hardware ethernet MAC;
  fixed-address NODE_IP;
}
<< done >>
```

**Comment by ColonelPanics** [comment]
Node DHCP entries could differ so a per-node dhcp template would be ideal, these could then all be merged to create the dhcpd.flight file.

Example - https://github.com/openflighthpc/flight-metal/blob/ce99638721fd7d01c3f9b808a59b50fafc95727a/examples/node/dhcp#L1-L4

Further functionality could provide domain-level default templates that are used if no node template is present.

**Comment by WilliamMcCumstie** [comment]
This was implemented in version `0.2.0`
